docs/products/dragonfly/get-started.md

```diff
@@ -1,23 +1,24 @@
 ---
 title: Get started with Aiven for Dragonfly®
+limited: true
 ---
 
 The first step in using Aiven for Dragonfly is to create a service. You
 can do so either using the [Aiven Console](https://console.aiven.io/) or
 the [Aiven CLI](https://github.com/aiven/aiven-client).
 
-:::important
-Aiven for Dragonfly® is currently a
-[limited availability](/docs/platform/concepts/beta_services) service. If you are interested in exploring this offering,
-reach out to our sales team at [sales@aiven.io](mailto:sales@aiven.io) for more information and
-access.
-:::
+## Prerequisites
+
+Aiven for Dragonfly® is currently a **limited availability** service and requires activation
+on your Aiven account. To explore this offering and request access for your account, contact our
+sales team at sales@aiven.io.
 
 ## Create a service using the Aiven Console
 
 1.  Log in to the [Aiven Console](https://console.aiven.io/).
-2.  Follow
-    [these instructions](/docs/platform/howto/create_new_service) to create a new Dragonfly service.
+1.  Follow
+    [these instructions](/docs/platform/howto/create_new_service) to create a new
+    Dragonfly service.
```

Comment:
Let's avoid generic phrasing like "click here <link>" etc, and reduce word count:

```suggestion
1.  [Create a Dragonfly service](/docs/platform/howto/create_new_service).
```